ui/lib/serverconfig.js

```diff
@@ -4,6 +4,11 @@ export function useServerConfig() {
   const { data: { isEmailConfigured, isSignupEnabled, baseDomain } = {} } =
     useSWR(`/api/server-configuration`, {
       revalidateIfStale: false,
+      onSuccess: (data) => {
+        if (data.baseDomain === "") {
+          data.baseDomain = window.location.host
```

Comment:
Great catch! This should live closer to where it's being rendered or used (vs mutating the data sent by the server).

Reply:
Updated the logic to just use the window location